pkg/jx/cmd/delete_context.go

```diff
@@ -7,7 +7,6 @@ import (
 	"strings"
 
 	"github.com/jenkins-x/jx/pkg/jx/cmd/templates"
-	"github.com/jenkins-x/jx/pkg/kube"
```

Comment:
Is this ok? I think `o.Kube().LoadConfig()` still needs this package.

Reply:
It compiles so I guess so!
`o.Kube().LoadConfig()` pulls in this package transitively I guess.